tests/performance/config/dcluster/large-cluster.conf

```diff
@@ -7,15 +7,18 @@ zookeeper = 3.4
 
 # Overcommit the resources, allocing deployment of at least 1000 apps
 # with the minimum requirements
-mesos_slaves = 6
-mesos_resources_cpus = 250
-mesos_resources_mem = 8192000
+mesos_slaves = 10
+mesos_resources_cpus = 500
+mesos_resources_mem = 67108864
 
 # Exposed ports
 marathon_port = 8080
 marathon_jmx = 9010
 mesos_master_port = 5050
 
 # Increase the number of instances launched per offer since the performance
-# tests should put pressure on marathon.
-marathon_args = "--max_instances_per_offer=500"
\ No newline at end of file
+# tests should put pressure on marathon. Also load the executorid plugin
+# from /opt/shared (the marathon-perf-testing infrastructure mounts there the
+# contents of the `files` folder from here
+# : https://github.com/mesosphere/marathon-perf-testing/tree/master/files )
+marathon_args = --max_instances_per_offer=1000 --launch_tokens=1000 --max_running_deployments=10000 --plugin_dir=/opt/shared/plugins-1.6 --plugin_conf=/opt/shared/executorid-plugin-config.json
```

Comment:
I would tune down the numbers. Otherwise, it's probably "too few" cycles for marathon to start all the tasks. How about `--max_instances_per_offer=100` and `--max_running_deployments=100`? I'm also not sure if `--launch_tokens=1000` has any real effect.

Reply:
1. `max_running_deployments` should be equal to the number of the instances to be deployed. This is a current limitation of the test, since it tries to submit them as quickly as possible and I would like to avoid re-tries in the scope of the test because the re-try interval is going to affect the results.

2. `--max_instances_per_offer=100` sure. I was just trying to minimise the deployment time.

3. It _does_ have an affect, but I think such a big value might have a negative effect. _Take the following with a grain of salt, since the data that I have collected are not big enough, but_ I do notice some tendency. For example, check this: 
![image](https://user-images.githubusercontent.com/883486/42169494-913e82a6-7e1c-11e8-9b0d-158c9334d0c6.png)

Reply:
Ok, got it. Lgtm